Index: manifest.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\r\n    \"display_name\": \"SillyTavern Outfit Engine\",\r\n    \"author\": \"Branden (Original by Lucanna)\",\r\n    \"version\": \"1.5.0-dev41\",\r\n    \"homePage\": \"https://github.com/BrandenStoberReal/ST-Outfits-Extended\",\r\n    \"js\": \"index.js\",\r\n    \"css\": \"style.css\",\r\n    \"loading_order\": 15,\r\n    \"generate_interceptor\": \"outfitTrackerInterceptor\"\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/manifest.json b/manifest.json
--- a/manifest.json	(revision e37df2af037b96f5d343f6e97208408c98d1f7aa)
+++ b/manifest.json	(date 1760132462801)
@@ -1,7 +1,7 @@
 {
     "display_name": "SillyTavern Outfit Engine",
     "author": "Branden (Original by Lucanna)",
-    "version": "1.5.0-dev41",
+    "version": "1.5.0-dev42",
     "homePage": "https://github.com/BrandenStoberReal/ST-Outfits-Extended",
     "js": "index.js",
     "css": "style.css",
